Source/Examples/WPF.SharpDX/CrossSectionDemo/MainWindow.xaml.cs

```diff
@@ -31,5 +31,10 @@ public MainWindow()
                 }
             };
         }
+
```

Comment:
This was a quick check to test the hit testing. I can move this to a new example or bind this to a command in the view model instead.